x/fees/client/cli/tx.go

```diff
@@ -0,0 +1,160 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	ethermint "github.com/tharsis/ethermint/types"
+
+	"github.com/tharsis/evmos/v3/x/fees/types"
+)
+
+// NewTxCmd returns a root CLI command handler for certain modules/erc20 transaction commands.
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "fees subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
+		NewRegisterDevFeeInfo(),
+		NewCancelDevFeeInfo(),
+		NewUpdateDevFeeInfo(),
+	)
+	return txCmd
+}
+
+// NewRegisterDevFeeInfo returns a CLI command handler for registering a
+// contract for fee distribution
+func NewRegisterDevFeeInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "register-fee [contract_hex] [withdraw_bech32]",
+		Short: "Register a contract for fee distribution",
```

Comment:
Add "Long" field. We should explain that the withdraw address will default to the deployer if it's not provided